README.md

```diff
@@ -284,11 +284,9 @@ For more git config ideas, and for credit for many of the aliases here, please s
   * <http://web.archive.org/web/20100911013406/http://stackoverflow.com/questions/267761/what-does-your-gitconfig-contain>
   * <http://web.archive.org/web/20111114085847/http://superuser.com/questions/169695/what-are-your-favorite-git-aliases>
   * <http://stackoverflow.com/questions/1309430/how-to-embed-bash-script-directly-inside-a-git-alias>
-  * <http://code.joejag.com/2013/everyday-git-aliases/>
   * <http://blog.apiaxle.com/post/handy-git-tips-to-stop-you-getting-fired/>
   * <https://ochronus.com/git-tips-from-the-trenches/>
   * <http://mislav.uniqpath.com/2010/07/git-tips/>
-  * <https://github.com/WuTheFWasThat/dotfiles/blob/master/.gitconfig>
```

Comment:
This one is not dead, just slightly changed: https://github.com/WuTheFWasThat/dotfiles/blob/master/gitconfig (remove the dot).

Reply:
Thank you for your review and you are right.
 I created a new PR #49 and fix it.